### PR TITLE
zh-translation: /docs/reference/glossary/mixer-handler.md 

### PR DESCRIPTION
**Pull request by huntsman-li**
ref: https://github.com/servicemesher/istio-official-translation

### CODE REVIEW

File: content/zh/docs/reference/glossary/mixer-handler.md [hunk 1]
```diff
@@ -1,7 +1,4 @@
 ---
 title: Mixer Handler
 ---
-
-Handlers represent fully configured Mixer adapters. A single binary adapter can be used
-with different configurations, each such configuration is known as a handler. At
-runtime, Mixer routes [instances](#mixer-instance) to one or more handlers.
+Handlers 相当于配置完备的 mixer 适配器。一个适配器二进制文件可以被不同配置使用，这些配置也可以称为 handler。在 mixer 运行时，Mixer 将 [instances](#mixer-instance) 路由到一个或多个 handlers。
```
Comment (rootsongjc) [inline content/zh/docs/reference/glossary/mixer-handler.md]: Handlers -> handler

Comment (huntsman-li) [reply content/zh/docs/reference/glossary/mixer-handler.md]: fix

File: content/zh/docs/reference/glossary/mixer-handler.md [hunk 2]
```diff
@@ -1,7 +1,4 @@
 ---
 title: Mixer Handler
 ---
-
-Handlers represent fully configured Mixer adapters. A single binary adapter can be used
-with different configurations, each such configuration is known as a handler. At
-runtime, Mixer routes [instances](#mixer-instance) to one or more handlers.
+handler 相当于配置完备的 mixer 适配器。一个适配器二进制文件可以被不同配置使用，这些配置也可以称为 handler。在 mixer 运行时，Mixer 将 [instances](#mixer-instance) 路由到一个或多个 handlers。
```
Comment (rootsongjc) [inline content/zh/docs/reference/glossary/mixer-handler.md]: 行首字母大写。

**Comment by huntsman-li** [comment]
fork代码勿删除，重新简历pr

**Pull request closed without merging**